lib/expectedConditions.js

```diff
@@ -0,0 +1,267 @@
+var webdriver = require('selenium-webdriver');
+
+/* globals browser */
+
+/**
+ * Represents a library of canned expected conditions that are useful for
+ * protractor, especially when dealing with non-angular apps.
+ * 
```

Comment:
Add some example usage here as well (especially using in the context of `browser.wait`)